tools/remove_html_comments_from_pr.py

```diff
@@ -0,0 +1,73 @@
+"""
+Edit pull request description to remove HTML comments
+
+We might want to remove section with markdown task lists that are completely empty
+"""
+
+import re
+import sys
+from os import environ
+
+import requests
+
+
+def remove_html_comments(text):
+    # Regular expression to remove HTML comments
+    html_comment_pattern = r"<!--(.*?)-->"
+    return re.sub(html_comment_pattern, "", text, flags=re.DOTALL)
```

Comment:
I guess if we really want, we can extract all the patterns, check that they are in the template, and remove them only if in the template ?

Reply:
I don't think it's that important or likely, so feel free to leave it, imo :)

Reply:
As I understand I will only catch single line comments?

Reply:
Should we add a newline to the end of the pattern? Without it, you get one newline per comment, which would be awkward in commit messages:

```python
In [1]: import remove_html_comments_from_pr as rhc

In [2]: descr = """
   ...: <!-- comment 0 -->
   ...: actual description
   ...: <!-- multiline
   ...: comment -->
   ...: <!-- comment 1 -->
   ...: """

In [3]: rhc.remove_html_comments(descr)
Out[3]: '\n\nactual description\n\n\n'
```

@Czaki it looks like multiline comments are removed ok. This appears to be because of the `DOTALL` flag. Also TIL about the "non-greedy modifier `?`" ([ref](https://docs.python.org/3/library/re.html#regular-expression-syntax)), which explains why the whole description isn't wiped out.

Reply:
Hm. Removed comments left empty lines. Did we want to remove them also?

Reply:
I'm on holidays for a week i'll update that when I'm back. Moving to draft, and will mark ready when back.